### DISCUSSION TITLE
感谢作者！能否增加一个功能：在书签里对一个分类里所有的标签自动全部连接。

### DISCUSSION THREAD

**Discussion by zhansimon** [electerm/electerm#3184] [General]
就是批量连接分类里的全部设备。

**Comment by zxdong262** [comment]
![Screen Shot 2023-12-21 at 07 30 58](https://github.com/electerm/electerm/assets/1641949/128f32ba-969c-4b95-90b6-ca366fda1bca)
已经有这个功能

**Reply by zhansimon** [reply]
感谢回复，辛苦了！加油！！！！
